views/signup.ejs

```diff
@@ -13,11 +13,16 @@
         <% }) %>    
     <% } %>
     <form action="/signup" method="POST">
-        <input type="text" name="userName" placeholder="User Name">
-        <input type="email" name="email" placeholder="Email">
-        <input type="password" name="password" placeholder="Password">
-        <input type="password" name="confirmPassword" placeholder="Confirm Password">
-        <input type="submit">
+        <input type="text" name="userName" placeholder="Enter Your User Name">
+        <input type="email" name="email" placeholder="Enter Your Email">
+        <input type="password" name="password" placeholder="Create Your Password">
+        <input type="password" name="confirmPassword" placeholder="Confirm Your Password">
+        <input type="text" name="firstName" placeholder="First Name">
+        <input type="text" name="lastName" placeholder="Last Name">
+        <input type="date" name="date" placeholder="Date Of Birth">
```

Comment:
lines 16-25:

Consider adding labels for fields. Particularly for the date as placeholder text doesn't appear for `<input type='date'>` fields.

E.g.,

```
<label for="date">Date of Birth</label>
<input type="date" name="date" placeholder="Date Of Birth">
```